d3servercheck.ps1

```diff
@@ -1,70 +1,52 @@
-function Write-Server-Result($tcpconns) {
+function Write-ServerResult($tcpconns) {
```

Comment:
Generally it's bad form to mix styling and content in PRs. Again I'm not averse to this change per se, but it is a tangent compared to the main goal.